impl/src/main/java/org/ehcache/impl/internal/store/loaderwriter/LocalLoaderWriterStore.java

```diff
@@ -63,6 +63,7 @@ public LocalLoaderWriterStore(Store<K, V> delegate, CacheLoaderWriter<? super K,
                                 ExpiryPolicy<? super K, ? super V> expiry) {
     this.delegate = delegate;
     this.cacheLoaderWriter = cacheLoaderWriter;
+
```

Comment:
Revert this if you have to update this PR again.

Reply:
done